src/mono/wasm/runtime/method-binding.ts

```diff
@@ -624,4 +624,4 @@ export type BoundMethodToken = {
     scratchBuffer: VoidPtr;
     scratchResultRoot: WasmRoot<MonoObject>;
     scratchExceptionRoot: WasmRoot<MonoObject>;
-}
\ No newline at end of file
+}
```

Comment:
maybe revert all the files that have whitespace damage at the end if nothing else changed... i'm not sure where this came from, maybe me originally?